torch/package/exporter.py

```diff
@@ -162,8 +164,15 @@ def save_source_string(self, module_name: str, src: str, is_package: bool = Fals
             for dep in dep_list.keys():
                 self.require_module_if_not_provided(dep)
 
-    def _import_module(self, module_name):
-        return import_module_from_importers(module_name, self.importers)
+    def _import_module(self, module_name: str):
+        try:
+            return import_module_from_importers(module_name, self.importers)
+        except ModuleNotFoundError as e:
+            if not is_mangled(module_name):
+                raise
+            msg = (f"Cannot export module '{module_name}'. Modules imported "
```

Comment:
Probably should still say "module not found". It's name has been mangled because it was loaded from a torch package. If you want to export it ...